client/ayon_core/hosts/maya/plugins/publish/validate_alembic_options_defaults.py

```diff
@@ -0,0 +1,94 @@
+import pyblish.api
+
+from ayon_core.pipeline import OptionalPyblishPluginMixin
+from ayon_core.pipeline.publish import RepairAction, PublishValidationError
+
+
+class ValidateAlembicOptionsDefaults(
+    pyblish.api.InstancePlugin, OptionalPyblishPluginMixin
+):
+    """Validate the attributes on the instance are defaults.
+
+    The defaults are defined in the project settings.
+    """
+
+    order = pyblish.api.ValidatorOrder
+    families = ["pointcache", "animation"]
+    hosts = ["maya"]
+    label = "Validate Alembic Options Defaults"
+    actions = [RepairAction]
+    optional = True
+
+    @classmethod
+    def _get_plugin_name(cls, publish_attributes):
+        for key in ["ExtractAnimation", "ExtractAlembic"]:
+            if key in publish_attributes.keys():
+                return key
+
+    @classmethod
+    def _get_settings(cls, context):
+        maya_settings = context.data["project_settings"]["maya"]
+        settings = maya_settings["publish"]["ExtractAlembic"]
+        return settings
+
+    @classmethod
+    def _get_publish_attributes(cls, instance):
+        attributes = instance.data["publish_attributes"][
+            cls._get_plugin_name(
+                instance.data["publish_attributes"]
+            )
+        ]
+
+        return attributes
+
+    def process(self, instance):
+        if not self.is_active(instance.data):
+            return
+
+        settings = self._get_settings(instance.context)
+
+        attributes = self._get_publish_attributes(instance)
+
+        msg = (
+            "Alembic Extract setting \"{}\" is not the default value:"
+            "\nCurrent: {}"
+            "\nDefault Value: {}\n"
+        )
+        errors = []
+        for key, value in attributes.items():
+            default_value = settings[key]
+
+            # Lists are best to compared sorted since we cant rely on the order
+            # of the items.
+            if isinstance(value, list):
+                value = sorted(value)
+                default_value = sorted(default_value)
+
+            if value != default_value:
+                errors.append(msg.format(key, value, default_value))
+
+        if errors:
+            raise PublishValidationError("\n".join(errors))
+
+    @classmethod
+    def repair(cls, instance):
+        # Find create instance twin.
+        create_context = instance.context.data["create_context"]
+        create_instance = None
+        for Instance in create_context.instances:
+            if Instance.data["instance_id"] == instance.data["instance_id"]:
+                create_instance = Instance
+                break
```

Comment:
Don't think make it uppercase first letter is a good python practice.
```suggestion
        for context_instance in create_context.instances:
            if context_instance .data["instance_id"] == instance.data["instance_id"]:
                create_instance = context_instance
                break
```

Or:
```suggestion
        create_instance = next((
            inst for inst in create_context.instances
            if inst.data["instance_id"] == instance.data["instance_id"]),
            None
        )
```

We could also remove the assert itself and instead just let `next()` error if we are pretty sure it _should_ exist.

```suggestion
        create_instance = next(
            inst for inst in create_context.instances
            if inst.data["instance_id"] == instance.data["instance_id"]
        )
```